app/editor/src/features/content/list-view/components/tool-bar/filter/InputOption.tsx

```diff
@@ -52,7 +52,7 @@ export const InputOption: React.FC<any> = ({
       <input type="checkbox" id={children} defaultChecked={isSelected} />
       <styled.SelectLabel
         className="label"
-        onClick={() => document.getElementById(children)?.click()}
```

Comment:
I was trying to figure out what was throwing errors, but I don't think this was it.  I think the drag+drop library is broken.